NEW_MODULE_TECH_EVAL.MD

```diff
@@ -0,0 +1,73 @@
+
+# New Module Technical Evaluation Processes
+
+## Overview
+This document outlines the processes for **technical** evaluation of module for inclusion in a FOLIO release.  The following activities are covered:
+1. Submission
+1. Evaluation
+1. Review
+1. Feedback
+1. Approval/Rejection
+
+### Supporting documentation
+* Details of the JIRA workflow are captured in a separate document:  (insert link). 
+* Technical evaluation is part of a larger process involving the other FOLIO Councils (PC/CC).  A description of the full process can be found here:  (insert link). 
+
+## Submission
+1. The Product Council fills out a form in JIRA (using a template) and submits it to the Technical Council.  
+    * See JIRA workflow for details.
+    * The Technical Council will only respond to submissions from the Product Council.
+1. The Technical Council will update the JIRA within 1 week (See JIRA workflow for details) with the following information:
+    * Confirm receipt of the submission
+    * Who will be performing the evaluation
+    * Provide an rough estimate for the initial evaluation to be completed
+        * Should be 3 weeks or less.
+    * Request additional information
+        * If there are missing points of contact, the Product Council will be contacted
+        * Any questions about the module will be directed to the points of contact.
```

Comment:
The framing statement of this list states that information will be shared with the submitter. I think that means some of the starts of the entries can be removed.

```suggestion
    * Receipt of the submission
    * Who will be performing the evaluation
    * Rough estimate for the initial evaluation to be completed
        * Should be 3 weeks or less.
    * Additional information needed
        * If there are missing points of contact, the Product Council will be contacted
        * Any questions about the module will be directed to the points of contact.
```